docs/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -88,6 +88,12 @@ This flag is only supported from the V2 version of the provider options struct w
 
 Default value: 0
 
+### enable_cuda_graph
+Check [using CUDA Graphs in the CUDA EP](../performance/tune-performance.md#Using-CUDA-Graphs-in-the-CUDA-EP) for details on what this flag does.
+This flag is only supported from the V2 version of the provider options struct when used using the C API. The V2 provider options struct can be created using [this](https://onnxruntime.ai/docs/api/c/struct_ort_api.html#a0d29cbf555aa806c050748cf8d2dc172) and updated using [this](https://onnxruntime.ai/docs/api/c/struct_ort_api.html#a4710fc51f75a4b9a75bde20acbfa0783). Please take a look at the sample below for an example.
```

Comment:
`Please take a look at the sample below for an example.`: I did not find an example of using `enable_cuda_graph` below. Do I miss something?

Reply:
You're right, removed it. We have linked to "Using-CUDA-Graphs-in-the-CUDA-EP" where the usage sample exists.